### PR TITLE
Bug 2005232: Pods list page should only show Create Pod button to user has sufficient permission

### PR DESCRIPTION
**Pull request by cyril-ui-developer**
### Before
 Screen Shot 2022-08-29 at 3 08 21 PM

### After
Screen Shot 2022-08-29 at 3 08 38 PM

### CODE REVIEW

**Comment by cyril-ui-developer** [comment]
Not ready for review

File: frontend/public/components/pod.tsx [hunk 1]
```diff
@@ -1021,13 +1021,20 @@ export const PodsPage: React.FC<PodPageProps> = ({
   const [data, filteredData, onFilterChange] = useListPageFilter(pods, filters, {
     name: { selected: [nameFilter] },
   });
-
+  const createNS = namespace || 'default';
+  const accessReview = {
+    groupVersionKind: referenceForModel(PodModel),
```
Comment (jhadvig) [inline frontend/public/components/pod.tsx:1026]: Lets call `referenceForModel(PodModel)` only once and use that variable in both places.

**Comment by cyril-ui-developer** [comment]
@jhadvig Updated, thanks.

**Review by jhadvig — APPROVED** [review]
/lgtm
/approve

**Pull request merged**